spec/models/order_cycle_spec.rb

```diff
@@ -392,6 +393,8 @@
     expect(occ.coordinator_fee_ids).not_to be_empty
     expect(occ.coordinator_fee_ids).to eq(oc.coordinator_fee_ids)
     expect(occ.preferred_product_selection_from_coordinator_inventory_only).to eq(oc.preferred_product_selection_from_coordinator_inventory_only)
+    expect(occ.schedule_ids).not_to be_empty
```

Comment:
As we test `expect(occ.schedule_ids).to eq(oc.schedule_ids)` just one line after, I guess this one is useless or may I miss something.

Reply:
You're not wrong, I was thinking the same. But I was just following the test pattern 😅. Maybe I should remove both instances of 'expect(...).to_not be_empty' from this spec?

Reply:
Oh ... I didn't see. Maybe it's ok to let it as it is. I don't see any contraindication.

Reply:
It's good to keep this line. Imagine the code would accidentally remove schedules from both order cycles. The next line would still pass but the result would be wrong.